transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/AddTransferMethodFragment.java

```diff
@@ -68,11 +70,13 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
+import java.util.Map;
 import java.util.TreeMap;
 
 public class AddTransferMethodFragment extends Fragment implements WidgetEventListener, AddTransferMethodContract.View {
 
-    public static final String TAG = AddTransferMethodFragment.class.getName();
+    protected static final String TAG = "transfer-method:add:collect-transfer-method-information";
+    private static final String LINK = "create-transfer-method";
```

Comment:
@peter-joseph This US3(PR-111) LINK constant name may conflict with US7(PR-112).